lib/services/build.ts

```diff
@@ -97,10 +97,11 @@ export class BuildService implements Project.IBuildService {
 	private requestCloudBuild(settings: Project.IBuildSettings): IFuture<Project.IBuildResult> {
 		return ((): Project.IBuildResult => {
 			settings.platform = this.$mobileHelper.normalizePlatformName(settings.platform);
-			let projectData = this.$project.projectData;
+			let projectData = this.$project.projectInformation.configurationSpecificData[settings.projectConfiguration.toLowerCase()];
```

Comment:
I think we'd rather we didn't support case sensitive configurations - I'll fix the code around so that we explicitly support case-insensitive configurations